### ISSUE TITLE
[Rendering] VoiceOver doesn't read the placeholder in TextView 

### ISSUE THREAD

**Issue by ZTY2016** [microsoft/AdaptiveCards#6703]
### Target Platforms

iOS

### SDK Version

1.5

### Application Name

Microsoft Teams

### Problem Description

1.Say "Hey Cortana" to enable Cortana
2.Say "send Message to someone(e.p. taoyuan)
3.Cortana will provide a TextView which placeholder is "Enter message", then using voiceover on the TextView

Bug:
VoiceOver only read "TextView. double tap to edit", the voice of placeholder on the textview is missing

### Card JSON

```JSON
none
```

**Comment by jonmill** [comment]
@ZTY2016 - is the application here Cortana or Teams?

**Comment by jonmill** [comment]
Closing due to lack of response
